ee/tabby-ui/lib/stores/exp-flag-store.ts

```diff
@@ -0,0 +1,15 @@
+import { EXPERIMENTAL_FALG } from '@/lib/constants'
+
+import { createStoreWithHydrated } from './utils'
+
+interface ExperimentalFlagInitialState {
+  [EXPERIMENTAL_FALG.QUICK_ACTION_BAR]: boolean
+}
+
+export const useExperimentalFlagStore =
+  createStoreWithHydrated<ExperimentalFlagInitialState>({
```

Comment:
it's fine to use zustand to create persistant store. But i generally feel it's more lightweight to use something like `useLocalStorage()` to wrap each individual flag as a pair of hook (plus global variable)

Reply:
I did a refactory, plz take a check